bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -263,6 +262,30 @@ public Map<String, Object> getDorProperties() {
         return customDorProperties;
     }
 
+    @JsonIgnore
+    public Map<String, String> visit(Function<FormComponent, Map<String, String>> callBack) throws Exception {
+        Map<String, String> initialResult = new HashMap<>();
+        return traverseChild(this, callBack, initialResult);
```

Comment:
we do we need a return type of Map<String, String>?
can we instead return void and let the caller handle the state?